README.md

```diff
@@ -2,30 +2,35 @@
 
 # NGINX Modern Reference Architectures
 
-A repository for modern application deployment and management, 
-created to illustrate modern app design principles and encourage 
-our community to deploy on Kubernetes. This project provides 
-a complete, stealable, easy-to-deploy, and standalone example 
-of how a modern app architecture can be built.
+A repository for modern application deployment and management, created 
+to illustrate modern app design principles and encourage our community 
+to deploy on Kubernetes. This project provides complete, stealable, 
+easy-to-deploy, and standalone examples of how a modern app architecture 
+can be built.
 
 
 
 ## Modern App Architectures
-While many different aspects of modern architectures exist, 
-we define them as driven by four fundamental characteristics: 
-*scalability*, *portability*, *resiliency*, and *agility*. 
 
-* **Scalability** – Quickly and seamlessly scale up or down to accommodate spikes or reductions in demand, anywhere in the world.
+While many different aspects of modern architectures exist, we define 
+them as driven by four fundamental characteristics: *scalability*, 
+*portability*, *resiliency*, and *agility*. 
 
-* **Portability** – Easy to deploy on multiple types of devices and infrastructures, on public clouds, and on premises.
+* **Scalability** – Quickly and seamlessly scale up or down to accommodate 
+spikes or reductions in demand, anywhere in the world.
 
-*	**Resiliency** – Can fail over to newly spun‑up clusters or virtual environments in different availability regions, clouds, or data centers.
+* **Portability** – Easy to deploy on multiple types of devices and 
+infrastructures, on public clouds, and on premises.
 
-*	**Agility** – Ability to update through automated CI/CD pipelines with higher code velocity and more frequent code pushes.
+*	**Resiliency** – Can fail over to newly spun‑up clusters or virtual 
```

Comment:
Is this still showing up as having a space between the bullet points and Resiliency and Agility? On my fork it doesn't look like this in plain text.